aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -3168,6 +3168,18 @@ swf:
   tags:
     product_categories: {}
   version: swf-2012-01-25
+synthetics:
+  api_ref: AmazonSynthetics/latest/APIReference/Welcome.html
+  blurb: ''
+  expanded:
+    long: Amazon CloudWatch Synthetics
+    short: CloudWatch Synthetics
+  long: '&CWlong; Synthetics'
+  short: '&CW; Synthetics'
+  sort: cwsynthetics
```

Comment:
This should be the short name minus AWS/Amazon because it's used to sort the TOC, which displays the short name.